### DISCUSSION TITLE
Morar de Aluguel

### DISCUSSION THREAD

**Discussion by manzoliric** [frontendbr/forum#1465] [Discussão]
Hello!

Estava querendo morar de aluguel para começar a ter um espaço mais meu, liberdade e tambem morar mais proximo ao trabalho, moro a quase 2h e é complicado.

Alguem aqui mora de aluguel e pode me falar como foi e como é a experiencia de se mudar? e se vale a pena?

Obrigado :)

**Comment by sbrj** [comment]
Não sei o que isso tem muito a ver com o github, mas vamos lá.

Tenho 31 e moro só, de aluguel a quase 5 anos. Na vida, a gnt cresce, majoritariamente, quando saímos da tal "zona de conforto". Geralmente, começamos a "dar valor" ao antigo lar, somente quando estamos em um novo, com novos desafios.

Prós:
"Supostamente" pode fazer o que quiser, desde que respeite os limites dos vizinhos e desde que seja no tempo que lhe sobrar, depois de realizar todas as tarefas diárias.

Contras:
Arrumar a casa;
Ir comprar a comida;
Fazer comida;
Lavar louça;
Lavar roupa;
Solidão em vários momentos;
Trabalhar;
Pagar as contas;
Ir ao médico sozinho;
Comprar os próprios remédios;
Pode ter certeza que esqueci de alguns itens ainda...

As pessoas acham que vc tem mais tempo, pq está morando só, mas vc passa a ter bem menos tempo, pois muitas responsabilidades surgem umas atrás das outras.

Aaaaaaa se fosse só a questão de "morar sozinho"... hehehe
E, claro, quando se tem $$$$$ de sobra, tudo fica um pouco mais fácil. Daí é só uma questão de organização, controlar o emocional e não ficar de mimimi.

**Comment by ViniciusGularte** [comment]
Opa tudo bem @manzolirch  ? Volta e meia to me mudando, como cresci no interior, era isto ou ficar de fora do mercado 😝 , já dividi casa com 15 pessoas (foi bem ruim), e atualmente divido um ap com um colega de trabalho,  além do que o @sbrj  disse, a solidão e tals , teu tempo vai ser mais curto, no meu caso dividindo a casa consigo dividir as tarefas e os gastos com meu colega, então o custo fica menor, mas vai fundo cara, liberdade é ótimo, posso me alevantar agora e ir pra aonde quiser sem ninguém me perguntando "aonde vc vai", eu gosto de cozinhar, posso me alimentar saudavelmente aqui, é só questão de controle emocional e bastante controle financeiro, as coisas podem quebrar de uma hora pra outra e você tem que estar preparado pra isso

**Comment by Roger-Melo** [comment]
@manzolirch Acho que isso aqui pode te dar alguns insights: https://youtu.be/eAFOnmmW7XM

**Comment by synga** [comment]
Cara me mudei muito com a família nos últimos anos, ainda não moro só e estou me planejando justamente baseado nos perrengues que passamos em casa.

Morar só tem um grande impacto tanto financeiramente quando emocionalmente/psicologicamente. Como já pontuaram a solidão e etc não vou focar nisso, apenas no financeiro.

Se você não for morar em uma casa já mobiliada tem que pensar nisso, comprar pelo menos o basico, então já tem que pensar em comprar uma geladeira, fogão, maquina de lavar, mesa, sofá talvez, um armário pra cozinha e contratar plano de telefone/internet.

Pra se mudar tem o carreto, tem empresas ou pessoas que só levam e você se vira sozinho pra carregar e descarregar o caminhão. Se forem coisas que não da pra levar montado é bom pensar num montador ou como você faria sozinho, tipo um guarda roupa.

Tem que pensar que morar só não tem ninguém pra te socorrer se der uma merda. Se for demitido, se quebrar algo, se surgir um imprevisto o bom é não correr atrás de pai e mãe, pode acontecer no começo? Claro, pode acontecer de você precisar ligar pra eles e "ow to precisando de uma grana aqui, tem como?", mas é bom não pensar nisso. Então tente reservar um dinheiro para pelo menos 6 meses de aluguel, agua, luz e internet caso aconteça um imprevisto e também deixar uns 3k pra eventuais imprevistos, quebrou algo, precisa de um pedreiro, furou algo, etc etc etc.

Também tem a escolha entre casa e ape. Casa normalmente é mais barato, tipo na região que eu moro uma casa com 3 cômodos (quarto, cozinha, sala, 1 banheiro, garagem) você acha num bairro legal por uns R$800, se for um apê do mesmo jeito você gasta o dobro.
Em casa como não moramos num bairro tão legal, pegamos uma casa e pagamos pra colocar uma concertina em volta e comprei uma daquelas câmeras ip de R$100 só pra assustar um pouco quem passa na frente.

Parte financeira acho que é isso que posso te falar. Boa sorte ai

**Comment by manzoliric** [comment]
@sbrj Acho que entra na questão do off-topic aqui do forum do frontendbr, nunca tinha pensando por esse lado haha realmente faz sentido, que como estarei sozinho, não terei tanto tempo, como tenho morando com minha mãe, obrigado pela visão.

@ViniciusGularte Realmente terei que ter muito controle, coisa que não tenho hoje, mas acho que serei obrigado a ter quando morar sozinho hahaha obrigado pelo comentário!

@Roger-Melo Cara, eu vi esse video ontem!!!! hahaha estava pesquisando no Youtube e faz muito sentido o que ela diz, fiquei bem animado quando vi esse video, obrigado!

@synga Obrigado tambem pela visão, realmente podem acontecer mil coisas e coisas bem complicadas, que talvez eu nunca tenha passado por isso sozinho.

**Comment by manzoliric** [comment]
Galera, pensei bastante e acho que irei dar esse passo, a minha situação hoje é bem "confortável", eu iria mudar de casa apenas para ter algo meu, não tenho nenhum problema em casa, me mudaria apenas para dar mais espaço para minha mãe.

Porém acabei de mudar de emprego, faz apenas 2 semanas, irei juntar dinheiro durante uns 3 meses, para me mudar no ano que vem, obrigado mesmo pela visão pessoal, me deu clareza em algumas coisas que eu nem sequer tinha pensado.

Obrigado! 💛

**Comment by felipefialho** [comment]
@manzolirch Fala mano, beleza?

Vou contar uma história bem pessoal aqui.

O ano era 2012, tinha acabado de fazer 22 anos. Meu pai estava próximo de se aposentar e tomou uma decisão: Iria se mudar para o interior. 

Nós morávamos na periferia de São Paulo e ele teria um salto enorme na qualidade de vida. Junto com isso, eu tomei a minha decisão: Ficaria em São Paulo.

Aluguei uma casa perto da onde morava, ainda na periferia, com o acesso difícil ao dia-dia da cidade, qualquer locomoção significava perder boa parte do meu dia, e o transporte público ainda era escasso. Mas é onde conseguiria pagar com o salário que ganhava.

Boa parte da minha renda (e as vezes toda ela ou mais que ela), ficava entre aluguel, alimentação, contas e manutenção da casa. Além disso, chegava em casa podre e ainda tinha que cuidar da limpeza, fazer comida, etc. E também tinha a solidão, de você chegar em casa e não ter ninguém, só você e seus pensamentos. Não foi fácil.

Mas ainda sim, foi a melhor e mais importante decisão que tomei na minha vida. Amadureci, ganhei casca, parei de me preocupar com besteira e passei a focar em coisas fundamentais. Minha visão de mundo mudou. Deixei de ser "moleque".

Sete anos se passaram, minha situação econômica foi melhorando e hoje consigo morar perto do trabalho (extremamente importante para uma vida equilibrada).

Então, é isso mano, boa sorte na tua decisão. E voa.

**Comment by kazzkiq** [comment]
Também estou com planos de morar sozinho de aluguel em breve e tô achando os relatos dessa Issue muito úteis!

Uma dúvida que tá me tirando o sono é o custo x benefício de morar perto do trabalho. Por um lado morar em bairros mais afastados significa menos custos com imóvel e opções melhores (mais espaço, etc), por outro lado a ideia de não pegar trânsito nenhum morando perto do serviço parece muuito interessante.

Mas ao mesmo tempo que vejo essa economia de tempo por morar perto, eu fico pensando se não acabo caindo no espiral de "viver para o trabalho": Eu já passo o dia fora de casa, aí alugo um lugar perto de onde trampo, pequeno, só pra passar a noite olhando pro teto e voltar pro trabalho no outro dia cedo. Sei lá, isso me preocupa um pouco.

A ideia de poder morar em um local afastado porém com mais qualidade de vida pelo mesmo preço (e com o ônus do trânsito) também acaba parecendo atrativa.

O que vocês acham que é mais vantajoso?

**Comment by MrS4w** [comment]
6 meses morando sozinho, realmente o que mais pesa é uma coisa chamada **responsabilidade** mas pra mim está valendo muito a pena porquê desde 2018 venho amadurecendo cada vez mais, mas se você conseguir se organizar certinho e ter um certo *controle financeiro* sempre pensando em **prioridades**, dá pra se virar tranquilamente, a sensação de não depender de ninguém, de liberdade e de ter um lugar pra chamar de seu (mesmo sendo aluguel) é inexplicável cara, de fato como citado acima, o pior inimigo é a solidão.

**Comment by felipefialho** [comment]
@kazzkiq Também rola fazer umas contas do custo com transporte e tempo gasto com transporte e ver se esse custo compensaria (se for possível, claro) uma mudança para próximo do trabalho.

Abordei muito desse assunto [nesse texto sobre valor hora](https://www.felipefialho.com/blog/2015/uma-reflexao-sobre-salarios-valor-hora-e-qualidade-de-vida), já antigo, mas ainda valido.

**Comment by tcelestino** [comment]
Depois de quase 10 anos que sair da casa de minha mãe e 6 anos morando em outra cidade, te digo que é a melhor experiência que você pode ter na sua vida. Seu senso de responsabiliade muda, passa a pensar mais na sua vida de hoje e do futuro e planejar tudo com muito mais eficiência.

Limpar casa, lavar os pratos, aprender fazer algo além do arroz e ovo frito, etc.., também é uma experiência. Dei muita sorte nesses quesitos, porque minha mãe me criou fazendo tudo isso desde pequeno, então limpar casa, lavar pratos, etc... pra mim é um prazer, ainda mais que faço a limpeza da casa aos sábados ouvindo música nas alturas. :laughing: 

Uma coisa que posso te dizer é que se você morar em SP, com certeza vai pagar caro para alugar um lugar que tenha um mais de estrutura (transporte, mercado, etc...). Se for de festa, precisa ganhar bem (muito bem mesmo) ou saber dosar as saídas. Acho que me viro bem nesse quesito, porque nunca fui tanto de curtir festas, porém chega uma hora da sua vida profissional e pessoal que você quer aproveitar, dai você precisa ter tudo mais planejado/organizado.

No mais, se tiver de boa nas finanças e tiver afim de uma nova percepção de vida, sempre é valido. 

:crab:

**Comment by fernandoRj** [comment]
eu daria a dica de vc juntar no minimo 1 ano de aluguel e deixar essa grana de reserva para emergência caso perca emprego nao precise passar por perrengues e depender de favores

**Comment by jpferreiradev** [comment]
Tópico muito foda e com ótimas dicas, eu estou passando seriamente em morar sozinho no ano que vem, ainda tenho sorte que na cidade aonde eu moro é pequena e os lugares não são longe e o preço dos alugueis está bem em conta, acho muito valido principalmente para o seu amadurecimento em todos os sentidos.

**Comment by gabriel-silveira** [comment]
Tópico interessante e correlato com nossa profissão.

Estou num momento em que, após várias mudanças de residência (a mais recente para perto da empresa em que fiquei por 2 anos), estou voltando a morar com minha família para ao invés de ficar gastando grana com aluguel, vou reduzir gastos, tentar investir e quem sabe adquirir um patrimônio em um futuro próximo. É claro que isso só é possível pois tenho família na mesma cidade em que trabalho (SP - capital). Para quem não tem essas condições, com certeza estar o mais próximo possível do local de trabalho é a melhor opção quando falamos em qualidade de vida, pois o deslocamento de casa para o trabalho é algo muito estressante.

**Comment by maucrvlh** [comment]
Essa thread me ajudou bastante.

Ainda sobre aluguel, qual local vocês indicam p/ aluguel em SP até 1.8k próximo ao Morumbi?

**Comment by kavalcante** [comment]
@maucrvlh Já chegou a ver na Vila Sônia?

**Comment by alvescaio** [comment]
@maucrvlh, estou a pouco mais de um mês em SP, morando no butantã, pago 1k e tô achando o bairro ótimo.

Caso não seja possível morar ao lado do trabalho, minha dica é procurar um lugar próximo a alguma estação de metrô.  No meu caso, moro próximo a estação butantã e trabalho na Paulista, então em menos de 30 min estou na empresa.

Sobre morar só, eu tenho 23 anos e desde 2014 moro sozinho. Sempre estou jogando online com meus amigos e isso ajuda bastante a 'dibrar' a solidão.

Quando morava com meus pais, eu tinha a sensação de que estava de carona e indo para um único lugar, sempre pelo mesmo caminho. Morar só, me deu maior autonomia sobre a vida. Mais liberdade de tocar meu caminho. A estrada que antes tinha somente uma direção, virou um cruzamentos de infinitos caminhos. O problema é que vc não está mais de carona com a sua família, vc desceu do carro e foi a pé.

**Comment by renanlbl** [comment]
Me identifiquei bastante com o amigo acima, tenho 23 anos e estou indo morar em SP, bairro Butantã, dia 02/03 ao dia 07/03 estarei lá procurando algo próximo ao metrô, pq de lá tem metro e trem pra toda cidade, (vou trabalhar na vila Olímpia) então se algum dev tiver com interesse em rachar um apartamento, entra em contato cmg... 21968701055

**Comment by maucrvlh** [comment]
Obrigado pelos dicas, @kavalcante e @alvescaio. ;)

**Comment by felipe-heredia** [comment]
Não falando como programador, nem como um "morador de aluguel", mas sim como estudante de economia e desenvolvimento pessoal.

_1.  Morar sozinho e de aluguel, pode te ajudar a ser mais produtivo. Ser melhor e ainda economizar mais grana.
2.  Você teria mais tempo sozinho para você, logo faria melhores as coisas, pois teria tempo pra pensar.
3.  Começaria a doer na pele, paga contas de água, luz, telefone etc._

Não moro de aluguel, por quê ainda não posso (apenas 15 anos), mas minha visão é essa, acordando a 5 da manhã, eu fico sozinho até umas 06:30, então consigo produzir mais, ler mais e fazer coisas melhores e pensar mais, pois não existe barulho algum e nenhuma chance de meus pais me pedirem pra que eu faça algo ou que conversem comigo.

### Essa é a minha visão, portanto, tire seus proveitos e analise seus pontos e como realmente eles são.

**Comment by eduardosoareszup** [comment]
> Não falando como programador, nem como um "morador de aluguel", mas sim como estudante de economia e desenvolvimento pessoal.
> 
> _1. Morar sozinho e de aluguel, pode te ajudar a ser mais produtivo. Ser melhor e ainda economizar mais grana. 2. Você teria mais tempo sozinho para você, logo faria melhores as coisas, pois teria tempo pra pensar. 3. Começaria a doer na pele, paga contas de água, luz, telefone etc._
> 
> Não moro de aluguel, por quê ainda não posso (apenas 15 anos), mas minha visão é essa, acordando a 5 da manhã, eu fico sozinho até umas 06:30, então consigo produzir mais, ler mais e fazer coisas melhores e pensar mais, pois não existe barulho algum e nenhuma chance de meus pais me pedirem pra que eu faça algo ou que conversem comigo.
> 
> ### Essa é a minha visão, portanto, tire seus proveitos e analise seus pontos e como realmente eles são.

Eu adicionaria um quarto ponto.
_4. Toda a manutenção da casa ou apartamento fica por sua conta, não tem sua mãe para ajudar na limpeza ou na comida e não tem seu pai para ajudar arrumar itens estragados, então é algo para se considerar.  Então nessa conta entra desde questão de tempo dedicado a isso, quanto a grana necessária para deixar tudo funcionando._

**Comment by eliseumds** [comment]
Vale. Você vai ter a liberdade de se mudar mais facilmente e rapidamente no futuro, caso precise, e isso pode acelerar muito o crescimento da sua renda. Eu moro de aluguel desde sempre e faço mais dinheiro com investimentos do que se tivesse comprado um imóvel, fora que eu não tenho dor de cabeça com manutenção, impostos, se eu não curtir o lugar eu saio, posso diminuir drasticamente o tempo perdido em transporte pro trabalho e etc. Mudei-me pra 3 países diferentes, e em dois dias me mudo de novo.

No começo pode ser difícil dependendo da cidade (eu gastava metade do salário em aluguel quando comecei em Sampa), mas o retorno pode ser muito maior também.

**Comment by citassp** [comment]
> Hello!
> 
> Estava querendo morar de aluguel para começar a ter um espaço mais meu, liberdade e tambem morar mais proximo ao trabalho, moro a quase 2h e é complicado.
> 
> Alguem aqui mora de aluguel e pode me falar como foi e como é a experiencia de se mudar? e se vale a pena?
> 
> Obrigado :)

Vale muito sim! Pra quem está saindo de casa pela primeira vez, hoje em dia já está bem mais fácil - tem muita opcao de ap mobiliado ou site que ajuda a encontrar roommate ou alguem pra dividir ap. Dependendo do orçamento, vale dar uma olhada no site do webquarto, housi, Yuca ou www.citas.com.br pra quem quer morar no centro de SP!
